TERMS.md

```diff
@@ -0,0 +1,121 @@
+	- An error is defined by referencing one or more **Type**s that define the data that shall be communicated when an error occurs.  Enumeration types are common, but it could be any defined type.
+	- There can be more than one error type defined for a method
+	- It is not specified by VSC how errors are propagated - this translation is target language/environment specific.
+		- It may be added as return/output parameters in one programming language and translated to **Exception**-handling in another.  Communication protocols may implement their own error-condition channels and approaches.
+
+## Target environment
+
+- This is a catch-all term to indicate the context and environment of those
+artifacts that are generated from source IDL.  
+- For each type of output stemming from a generator or conversion tool, there
+will be environment-specific details to consider, and in the documentation we
+often refer to all of those as simply the "target environment".
+- The term thus signifies any and all things that are unique about that environment.
+
+For example:
+ - The chosen programming language in the case of code-generation tools.
+ - Details pertaining to other levels of software technology that is being
+ targeted.  For example if a particular protocol is applicable in the output,
+ such as HTTP, details of transport layer security (TLS) would apply, whereas
+ for other protocols it might not.
+
+N.B. In the text, "target-environment dependent", may sometimes be shortened to simply target-dependent.
+
+----
+
+NEW 2022-08-08:
+
+----
+
+## Properties
+
+(N.B. In Franca IDL these are called Attributes)
+
+- A **Property** is an observable data item.  It belongs in a **Namespace**, has a canonical name in addition to possible **aliases** (i.e. alternative names), and a data type.
+- A Property is defined as a single item but the data type could be an arbitrary data type, including composite data types.
+- The singular nature of a property suggests that its value shall updated and communicated as an atomic operation, i.e. it is either fully updated or not at all.
+
+- The method of update and communication is typically target-environment specific, but the most common expectation is that a Property can be updated and _observed_, meaning that a client can call a method to get or set the current value of the property (with appropriate access control rules) and often also _subscribe_ to be notified of when the value of a property was changed for any reason.
+- In some other language definitions this type of item is called an **Attribute** or **Field**.
+- A property can also be seen as analogous (and bi-directionally translatable, if datatypes allow it) to a Sensor Signal in the Vehicle Signal Specification (VSS).
+
+**Side proposal**: Consider renaming Property to Attribute to match Franca IDL, and because properties have another use in OpenAPI and AsyncAPI, and it seems other HTTP-related communication as well.
```

Comment:
But in e.g. [SSN/SOSA](https://www.w3.org/TR/vocab-ssn/#SOSAActuatableProperty) the term Property is used. Perhaps it would be an idea to have a table to compare VSC properties (or if we call them attributes) with similar definitions in Franca/SOSA/VSC.

Reply:
Thanks for noting.  Yes, there are a lot of different things to compare to make an informed choice since different systems use the terms differently.